engine/crates/engine/src/types/merged_object.rs

```diff
@@ -50,26 +50,26 @@ where
     fn create_type_info(registry: &mut Registry) -> crate::registry::MetaFieldType {
         registry.create_output_type::<Self, _>(|registry| {
             let mut fields = IndexMap::new();
-            let mut cc = CacheControl::default();
+            let _cc = CacheControl::default();
 
             if let MetaType::Object(ObjectType {
                 fields: b_fields,
-                cache_control: b_cc,
+                // cache_control: b_cc,
```

Comment:
Are changes in this file as intended? Are there consequences to not merging the cache control there?

Reply:
Yes, as intended.  This is part of the original async_graphql codebase, not in the codepath for production, I'm not wasting my saturday keeping it right 😆

Reply:
Ah good, if it's async_graphql stuff we don't use that makes complete sense.